resources/migrations/20181231124500-option-form-item.up.sql

```diff
@@ -0,0 +1,15 @@
+-- :disable-transaction
+ALTER TYPE itemtype
+  ADD VALUE 'option';
+--;;
+CREATE TABLE application_form_item_options
```

Comment:
I wonder if it would actually make more sense to start moving these towards document orientation as well? I.e. store the heterogenous options in the item rows.

Reply:
That would be good. It was a pain to implement this relationally.

We'll need to discuss this refactoring more. We could add a JSON field to application_form, or as a smaller step add it to application_form_item.